src/elarm_alarmlist.erl

```diff
@@ -151,36 +151,36 @@ al_test_() ->
      ]}.
 
 just_started() ->
-    {ok,State} = init([]),
-    ?assertEqual({{ok,[]}, State}, get_alarms(State)),
-    EvtId = erlang:now(),
-    ?assertEqual({{error, not_active},State}, get_alarm(EvtId, State)),
-    ?assertEqual({{error, not_active},State}, get_alarm(full, disk_1, State)).
+    {ok, State} = init([]),
+    ?assertEqual({{ok, []}, State}, get_alarms(State)),
+    EvtId = erlang:timestamp(),
```

Comment:
As for now, OTP 18.2 still supports erlang:now (even thought it is deprecated), so this is not something I would address in this PR. 

As for the future, UUID would be more suitable than erlang:timestamp (which doesn't generate unique values as far as I understand). You are welcome to create a separate PR about that, e.g. by using http://github.com/travis/erlang-uuid.